frontend/src/components/elements/service/index.jsx

```diff
@@ -43,7 +43,7 @@ export const Service = ({
                     target={"_blank"}
                     rel="noopener noreferrer"
                 >
-                    <DigitText.Heading5 text={title} bold />
+                    <DigitText.Heading5 text={title} bold="true" />
```

Comment:
I'm pretty sure this is the same thing

Reply:
But.. a string?

Reply:
Yeah I don't know, I noticed this error message in the console so I thought I'd change it.
```
Warning: Received 'true' for a non-boolean attribute 'bold'
If you want to write it to the DOM, pass a string instead: bold="true" or bold={value.toString()}.
```